gpkitmodels/GP/aircraft/wing/sparloading.py

```diff
@@ -65,22 +62,19 @@ def setup(self, wing, state, sp=False):
         sigma = self.wing.spar.material.sigma
         deta = self.wing.planform.deta
 
-        if sp:
-            with SignomialsEnabled():
-                shear = S[:-1] >= (S[1:] + 0.5*deta*(b/2.)*(q[:-1] + q[1:])
-                                   - Sout)
-        else:
-            shear = S[:-1] >= S[1:] + 0.5*deta*(b/2.)*(q[:-1] + q[1:])
+        constraints = []
+        if not sp:
```

Comment:
the `sp` argument appears to be unused now?

Reply:
Well, in the solar SP model I can't have those constraints appear so... that why the `sp` arg is still there.